src/App.scss

```diff
@@ -223,14 +223,18 @@ h6 {
   flex-direction: row;
   justify-content: space-around;
   flex-wrap: wrap;
-  margin-left: 10rem;
-  margin-right: 10rem;
+  margin-left: 8rem;
+  margin-right: 8rem;
```

Comment:
Is this change even necessary with the margin-left ?

Reply:
Yes, we need to decrease the margin from home else how would we give margin-left as there is no space to accommodate margin-left

Reply:
Did you check it? From what I see it's not necessary.

Reply:
Yes, I have checked it